src/DataLayer/KeyMotorData/KeyMotorData.cpp

```diff
@@ -21,19 +21,21 @@ void KeyMotorData::setMotorZero(KeyMotor motorZero)
 {
     motorZero_ = motorZero;
     emit motorZeroReceived(motorZero_);
-    emit motorSetPowerReceived(((motorZero_.busVoltage() + motorOne_.busVoltage()) / 2) * ((motorZero_.busCurrent() + motorOne_.busCurrent()) / 2));
```

Comment:
Since this is being removed, we should remove it from `I_KeyMotorData.h`, as well as `KeyMotorPresenter.cpp/.h`